app/controllers/comments_controller.rb

```diff
@@ -3,11 +3,11 @@ def create
     user = current_user
     @comment = Comment.new(
       text: params[:text],
-      posts_id: params[:posts_id],
+      post_id: params[:post_id],
       author_id: user.id
```

Comment:
- [ OPTIONAL ] I highly recommend to create methods with strong parameters. I assign it as optional because the previous reviewer didn't mention it or mention it but he put the wrong screenshot since you should create the params method like this


```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```